.travis.yml

```diff
@@ -5,12 +5,12 @@ matrix:
   include:
     - python: 3.6
       dist: trusty
-      env: TOXENV=py36 NO_TESTS_OVER_WIRE=1
+      env: TOXENV=py36 NO_CREDENTIALS_TESTS=1
       sudo: false
     - python: 3.7
       # from https://github.com/travis-ci/travis-ci/issues/9815#issuecomment-401756442
       dist: xenial
-      env: TOXENV=py37 NO_TESTS_OVER_WIRE=1
+      env: TOXENV=py37 NO_CREDENTIALS_TESTS=1
```

Comment:
We can see how these go, and tweak if this is flaky.

I was envisioning a py36 unittest run, a py37 unittest run, and an integration/network test run with live integration secrets, esp. once we switch our CI tests in this repo to taskcluster. That should help give us more information, while isolating the network tests' variability from the local tests.

Reply:
Hm, and it broke on this PR :) Maybe we should either enable in a 3rd non-blocking test, or keep them disabled in Travis til we have a better solution?

Reply:
I've done the former.